cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSender.java

```diff
@@ -475,7 +475,12 @@ else if (getProperties) {
 	}
 
 	private Message sendMessageForActionCreate(Session cmisSession, Message message, PipeLineSession session) throws SenderException, TimeOutException {
-		String fileName = (String) session.get(getFilenameSessionKey());
+		String fileName = null;
+		try {
+			fileName = session.getMessage(getFilenameSessionKey()).asString();
+		} catch (IOException e) {
+			throw new SenderException("Unable to get filename from pipeline session");
```

Comment:
This is not entirely correct, `fileName` may be empty. Perhaps you could do a `if(Message.isNotEmpty(...` first before trying to parse a potential `null` as String